tests/drivers/can/canfd/src/main.c

```diff
@@ -345,7 +345,7 @@ static void send_receive(const struct zcan_filter *filter1,
  * This test case must be run before sending/receiving test cases as it allows
  * these test cases to send/receive their own frames.
  */
-static void test_set_loopback(void)
+ZTEST(canfd_driver, test_set_loopback)
```

Comment:
The test depends on `test_set_loopback` being executed first, but, as tests are now per default executed in alphanumerical, this is not the case.
I am not sure if this could be executed as a setup function if includes a assert?

Reply:
Agreed.

Reply:
change the test_set_loopback() to be part of setup function.